Common/src/config_structure.cpp

```diff
@@ -1527,6 +1527,9 @@ void CConfig::SetConfig_Options(unsigned short val_iZone, unsigned short val_nZo
   /*!\brief WRT_CSV_SOL
    *  \n DESCRIPTION: Write a surface CSV solution file  \ingroup Config*/
   addBoolOption("WRT_CSV_SOL", Wrt_Csv_Sol, true);
+  /*!\brief WRT_CSV_SOL
+   *  \n DESCRIPTION: Write a binary coordinates file  \ingroup Config*/
```

Comment:
What is the intended use of this binary file ?

Reply:
Since the deformation algorithm only changes the x,y,z coordinates of the nodes in a mesh and not the connectivity, this file makes it easy to use SU2_DEF as a stand-alone mesh deformation tool by exporting the coords to a binary file for use downstream.